config/drupal/phpcsfixer.rules.yml

```
@@ -85,11 +85,9 @@ parameters:
                 - switch
                 - throw
```

should add  `curly_brace_block`

```suggestion
                - throw
                - curly_brace_block
```

Otherwise function will keep extra spaces (see fail on tests)

---

Added